VERSION

```diff
@@ -1 +1 @@
-1.0.0
+1.0.1
```

Comment:
So we are not moving to 2.0.0 but bumping up minor version for every commit?

Reply:
Latest change is 1.0.1.
Yes. This we decided in a team discussion.

Reply:
Okay.